test_scripts/API/VehicleData/HandsOffSteering/common.lua

```diff
@@ -0,0 +1,288 @@
+---------------------------------------------------------------------------------------------------
+-- Common module
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local actions = require("user_modules/sequences/actions")
+local json = require("modules/json")
+local utils = require("user_modules/utils")
+local runner = require('user_modules/script_runner')
+local SDL = require("SDL")
+
+--[[ General configuration parameters ]]
+config.application1.registerAppInterfaceParams.syncMsgVersion.majorVersion = 6
+config.application1.registerAppInterfaceParams.syncMsgVersion.minorVersion = 2
+config.defaultProtocolVersion = 2
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local m = {}
+local hashId = {}
+local handsOffSteeringResponseData = {
+  dataType = "VEHICLEDATA_HANDSOFFSTEERING",
+  resultCode = "SUCCESS"
+}
+
+--[[ Shared Functions ]]
+m.Title = runner.Title
+m.Step = runner.Step
+m.getPreloadedPT = actions.sdl.getPreloadedPT
+m.setPreloadedPT = actions.sdl.setPreloadedPT
+m.registerApp = actions.registerApp
+m.registerAppWOPTU = actions.registerAppWOPTU
+m.activateApp = actions.activateApp
+m.getMobileSession = actions.getMobileSession
+m.getHMIConnection = actions.getHMIConnection
+m.getParams = actions.app.getParams
+m.cloneTable = utils.cloneTable
+m.start = actions.start
+m.postconditions = actions.postconditions
+m.policyTableUpdate = actions.policyTableUpdate
+m.getAppsCount = actions.getAppsCount
+m.EMPTY_ARRAY = json.EMPTY_ARRAY
+m.deleteSession = actions.mobile.deleteSession
+m.connectMobile = actions.mobile.connect
+m.wait = utils.wait
+m.spairs = utils.spairs
+
+--[[ Common Functions ]]
+--[[ @updatedPreloadedPTFile: Update preloaded file with additional permissions for handsOffSteering
+--! @parameters:
+--! pGroup: table with additional updates (optional)
+--! @return: none
+--]]
+function m.updatedPreloadedPTFile(pGroup)
```

Comment:
@GetmanetsIrina I would propose to rename this function to `updatePreloadedPTFile`